examples/src/test/java/io/grpc/examples/routeguide/RouteGuideServerTest.java

```diff
@@ -85,12 +85,8 @@ public void setUp() throws Exception {
   }
 
   @After
-  public void tearDown() {
-    try {
-      server.stop();
-    } catch (InterruptedException e) {
-      e.printStackTrace();
-    }
+  public void tearDown() throws Exception {
```

Comment:
Couple of points:
- when Exception does happen, it will fail the test (before it would have passed). I think that's better but making sure that's what you want.

- `throws` is wider than it needs to be (`InterruptedException`).

Reply:
Yes, I want the test to fail. Yes, I purposefully used Exception, since that is pretty normal in tests since most exceptions should just cause the test to fail.